pallets/offchain-worker/Cargo.toml

```diff
@@ -6,30 +6,31 @@ homepage = 'https://litentry.com'
 license = 'Unlicense'
 name = 'pallet-offchain-worker'
 repository = 'https://github.com/litentry/litentry-node'
-version = '2.0.1'
+version = '3.0.0'
 
 [package.metadata.docs.rs]
 targets = ['x86_64-unknown-linux-gnu']
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "1.3.4", default-features = false }
+codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
 serde = { version = "1.0.119", optional = true }
-sp-core = { version = "2.0.1", default-features = false }
-sp-io = { version = "2.0.1", default-features = false }
-sp-std = { version = "2.0.1", default-features = false }
-alt_serde = { version = "1.0.104", default-features = false, features = ["derive"] }
+sp-core = { version = "3.0.0", default-features = false }
+sp-io = { version = "3.0.0", default-features = false }
+sp-std = { version = "3.0.0", default-features = false }
+alt_serde = { version = "=1.0.104", default-features = false, features = ["derive"] }
 serde_json = { version = "1", default-features = false, git = "https://github.com/Xanewok/json", branch = "no-std", features = ["alloc"] }
 
 parking_lot = "0.10.0"
-sp-runtime = { default-features = false, version = '2.0.1' }
-frame-support = { default-features = false, version = '2.0.1' }
-frame-system = { default-features = false, version = '2.0.1' }
-sp-arithmetic = { default-features = false, version = "2.0.1" }
+sp-runtime = { default-features = false, version = '3.0.0' }
+frame-support = { default-features = false, version = '3.0.0' }
+frame-system = { default-features = false, version = '3.0.0' }
+sp-arithmetic = { default-features = false, version = "3.0.0" }
 account-linker = {path = "../account-linker", package = "pallet-account-linker", default-features = false}
-frame-benchmarking = {default-features = false, version = '2.0.1', optional = true}
+frame-benchmarking = {default-features = false, version = '3.0.0', optional = true}
 
 [dev-dependencies]
-pallet-balances = { default-features = false, version = '2.0.1' }
+pallet-balances = { default-features = false, version = '3.0.0' }
+#sp-keystore = { default-features = false, version = '0.9.0' }
```

Comment:
Commented because test is commented out?

Reply:
Yes. This could be useful for ocw testing but not sure.